bundles/core/org.openhab.core.jsr223/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.bundles</groupId>
+    <artifactId>core</artifactId>
+    <version>1.7.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <bundle.symbolicName>org.openhab.core.jsr223</bundle.symbolicName>
+    <bundle.namespace>org.openhab.core.jsr223</bundle.namespace>
+  </properties>
+
+  <groupId>org.openhab.core</groupId>
+  <artifactId>org.openhab.core.jsr223</artifactId>
+
+  <name>openHAB Core Jsr223</name>
+  <packaging>eclipse-plugin</packaging>
+
+	<build>
+	  <resources>
+    	<resource><directory>src</directory></resource>
+  	</resources>
+	</build>
```

Comment:
please add properties for jdeb and jdeb dependency in order to get this bundle being exported as deb package as well